README.md

```diff
@@ -42,7 +42,7 @@ You need to install and configure this first:
    If you use a specific port with PostgreSQL, you may also need to set
    `$db['default']['port'] = "5432";`
 
-1. Import gammu database schema (it's included on gammu source, eg. `gammu/docs/sql/mysql.sql`)
+1. Import gammu database schema (it's included on gammu source, eg. `gammu/docs/sql/mysql.sql`) or download from `https://raw.githubusercontent.com/gammu/gammu/master/docs/sql/mysql.sql`
```

Comment:
I understand that it's useful to have such a link for reference, but I don't think it's a good idea to link the master branch of upstream. Nothing guaranties from a change in the schema and that the file is in line with the version of gammu-sms the user installed. It's better to redirect the user to the sources he used or the packaging system he uses.

Reply:
Aggred! Removing this.